### DISCUSSION TITLE
k3s k3s version v1.19.5+k3s2 (746cf403)

### DISCUSSION THREAD

**Discussion by pitosalas** [k3s-io/k3s#10254] [General]
I have a cluster that has been running fine with k3s version v1.19.5+k3s2 (746cf403) on three Ubuntu 20.04 servers. It is used by students at Brandeis University to run individual ROS (Robot OS) environments. As far as I can tell things are fine. Yet my version of K3S is old. I am about to add a new node so it's time to look at versions. Should I update?

**Comment by cwayne18** [comment]
Yes.

**Reply by pitosalas** [reply]
Ok. I left this out: I am reluctant to mess with a working environment unless there's a strong reason. Can you say what my risk is leaving the old version in place?

**Reply by cwayne18** [reply]
Generally speaking, using 4 year old un-patched versions is a pretty massive security risk, plus with versions that old the documentation can be a bit out of date, which could very easily cause confusion when the docs reference a 2 year old feature that is still 2 years newer than the version you're running.
